common/views/components/OptimalSort/OptimalSort.js

```diff
@@ -0,0 +1,7 @@
+const OptimalSort = () => (
+  <div dangerouslySetInnerHTML={{__html: `
+    <!-- BEGIN Optimal Workshop Intercept Snippet --><div id='owInviteSnippet' style='position:fixed;right:20px;bottom:20px;width:280px;padding:20px;margin:0;border-radius:6px;background:#333333;color:#EEEEEE;text-align:left;z-index:2200000000;opacity:0;transition:opacity 500ms;-webkit-transition:opacity 500ms;display:none;'><div id='owInviteMessage' style='padding:0;margin:0 0 20px 0;font-size:16px;'>Got a spare 3 minutes to help us improve our website?</div><a id='owInviteOk' href='https://t511eh14.optimalworkshop.com/optimalsort/wellcome1?tag=intercept&utm_medium=intercept' target='_blank' style='color:#7DB6EE;font-size:16px;font-weight:bold;text-decoration:underline;'>I&#x27;ll do it</a><a id='owInviteCancel' href='javascript:void(0)' onclick='this.parentNode.style.display="none";' style='color:#EEEEEE;font-size:14px;text-decoration:underline;float:right;'>No thanks</a></div><script>var owOnload=function(){if(-1==document.cookie.indexOf('ow-intercept-optimalsort-iz1q8471')){var o=new XMLHttpRequest;o.onloadend=function(){try{var o=document.getElementById('owInviteSnippet');this.response&&JSON.parse(this.response).active===!0&&(document.cookie='ow-intercept-optimalsort-iz1q8471=Done',setTimeout(function(){o.style.display='block',o.style.opacity=1},2e3))}catch(e){}},o.open('POST','https://www.optimalworkshop.com/survey_status/optimalsort/iz1q8471/active'),o.send()}};if(window.addEventListener){window.addEventListener('load',function(){owOnload();});}else if(window.attachEvent){window.attachEvent('onload',function(){owOnload();});}</script><!-- END Optimal Workshop snippet -->
+    `}} />
+);
+
```

Comment:
We could turn this into a component with a `componentDidMount`?
Type safety etc etc